### PR TITLE
FSF-2672 Updated master class diagram

### PR DESCRIPTION
**Pull request by yvesb04**

### CODE REVIEW

File: docs/master-overview.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+![ASB Continuous Monitoring Sequence](./assets/master-overview/ASB%20Continuous%20Monitoring%20Flowchart.jpg)
+
+## Main Loop Sequence
+### Set-up
+Before calculating the state, we need to define some CAN callbacks. They are used to receive RES signals, brake pressure, TS state, wheel information, mission information, emergencies, and timestamps (the computing unit, inversor, and steering pcb all need to send "alive" signals at a fixed rate, to confirm they operational). Those  callbacks are asynchronous, which means that they will shortly interrupt the main loop when data from a topic the master node is subscribed to is received to update the relevant variables. We also need to define the operation modes of all the pins that we will be sending and receiving information from.
+### Loop
+During the main loop, we first read all the values from the necessary components (Sensors, AMI, Switches, SDC Logic, EBS, ASSI). After this we calculate and update the state through a sequence of monitoring sequences, updating components' variables as depicted in the following diagram. After this, state and other variables are sent out to the rest of the system.
```
Comment (marhcouto) [inline docs/master-overview.md:26]: AMI and ASSI are not reads

File: docs/master-overview.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Master PCB
+
+The Master PCB is responsible for calculating and publishing the AS state that the car should be in and performing all the necessary actions required to achieve it. The state is correctly identifiable by the following flowchart, as required by rule **T14.9.2**: "*The status of the AS must be determined according to the flowchart in Figure 17.*
+
+![ASSFlow](./assets/master-overview/ASS-flow.png)
+
+Although the above diagram can determine the state, **it is only meant for externally determining the ASState**, it is **not** how the state is calculated.
```
Comment (marhcouto) [inline docs/master-overview.md:7]: Justify it (as it leaves a lot of undefined transitions in our system)

File: docs/master-overview.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Master PCB
+
+The Master PCB is responsible for calculating and publishing the AS state that the car should be in and performing all the necessary actions required to achieve it. The state is correctly identifiable by the following flowchart, as required by rule **T14.9.2**: "*The status of the AS must be determined according to the flowchart in Figure 17.*
```
Comment (marhcouto) [inline docs/master-overview.md:3]: Maybe make an example of the actions taken, just to clarify

**Review by marhcouto — CHANGES_REQUESTED** [review]
Notes left on the README. Diagrams are good. Waiting for the other one

File: docs/master-overview.md (hunk 1 above)
Comment (yvesb04) [reply docs/master-overview.md]: fixed, but from where does the Mission read come from then?

**Review by marhcouto — APPROVED** [review]
A cena da justificação que te falei e so reparei agr, muda o .md para ser README

**Comment by andrelds11** [comment]
Updated diagrams. Just one question. In the eletrical integration diagram, the Steer Alive should just be the Steering Angle? We're doing it like that in the code

**Pull request merged**